src/styles/utilities.css

```diff
@@ -59,6 +59,7 @@
   top: 150%;
   left: 50%;
   margin-left: -70px;
+  pointer-events: none;
```

Comment:
fixes bug where tooltip appears when hovering on hidden tooltip text (since we set `opacity` and not `visibility`, tooltip text is still hoverable when hidden)